ga.rpmtw.rpmlauncher.yml

```
@@ -0,0 +1,39 @@
+app-id: ga.rpmtw.rpmlauncher
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: RPMLauncher
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

Correct me if I'm wrong, but Minecraft itself isn't working on Wayland natively?